src/components/questionTwo/QuestionTwo.js

```diff
@@ -0,0 +1,355 @@
+import React from 'react';
+import PageContainer, { Section, Image } from '../pageContainer/PageContainer';
+
+const Reference = ({ id, text }) => (
```

Comment:
This is a private functional component, I just used it so the reference list wouldn't be spammed with repetitive style tags.